tutorial/openEO_intro/Agenda_and_Links.md

```diff
@@ -0,0 +1,42 @@
+# Part-2: OpenEO (Comment: Intro Session)
+
+This Part is going to introduce participants to openEO Platform. Attendees will learn what openEO Platform is, how to find data and run first process graphs. 
+We will also show attendees how to integrate custom code into your workflow using User Defined Functions.
+
+## 11:00 Getting started with OpenEO (15 minutes) 
+Benjamin (EODC)
+
+- Intro Presentation and Questionnaire on who is the audience
+
+
+## 11:15 Finding Data, Running first graphs, difference to client-side processing
+
+Daniel (Sinergise)
+- Logging in People and showing them around
+- Showing Different Clients (R, Python & Web editor)
+- Show viewing functionality in webeditor  
+
+-> switch to python client here
+
+Valentina/Sean (EODC)
+
+- Notebook showing how to search data & find metadata
+- Aggregate temporal period, Corine Landcover - Change detection , Tone Mapping
+- Local/ Client Side Processing: → Align with Michele on this what to show - replicate a small UC that we are presenting anyways
+
+
+## 11:45 Integrate custom code into your workflow using User Defined Functions 
+Jeroen (VITO)
+ 
+- Take over logged-in people showing them workflow creation and custom code introduction into their workflow 
```

Comment:
I would remove the "take over logged-in people showing..." (this is a note for us which should not be part of our public agenda. I would actually remove this whole bullet as it's included in the next two.